argilla-server/src/argilla_server/apis/routes.py

```diff
@@ -114,6 +115,8 @@ def create_api_v1():
     # We keep the same error responses as the API v0 for the moment
     APIErrorHandler.configure_app(api_v1)
```

Comment:
Since I see this line. Should we add the new `UnprocessableEntityError` to the `APIErrorHandler.configure_app` method and review the new code format during the v2 endpoints migrations?

Reply:
I think we should try to remove `APIErrorHandler` and try to do something like I'm doing with `add_exception_handlers_v1`. Not sure if that's a lot of work for the migration and it's something we should do in small steps.